select-conditionally/action.yml

```
@@ -0,0 +1,34 @@
+name: Select value based on condition
+description: 'Select value based on condition'
+
+inputs:
+  condition:
+    description: 'Condition for value selection. Possible values: [true, false]'
+    required: true
+  if_condition_true:
+    description: 'Value to return if condition true'
+    required: true
+  if_condition_false:
```

```suggestion
  if_true:
    description: 'Value to return if condition true'
    required: true
  if_false:
```

It's a matter of taste, but I would have shortened this as above.